src/CTA.Rules.Actions/ClassActions.cs

```diff
@@ -152,5 +160,33 @@ public class ClassActions
             };
             return RenameClass;
         }
+        public Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> GetReplaceMethodModifiersAction(string methodName, string modifiers)
+        {
+            Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> ReplaceMethodModifiers = (SyntaxGenerator syntaxGenerator, ClassDeclarationSyntax node) =>
+            {
+                var allMembers = node.Members.ToList();
+                var allMethods = allMembers.OfType<MethodDeclarationSyntax>();
+                if(allMethods.Count() > 0)
+                {
+                    var replaceMethod = allMethods.Where(m => m.Identifier.ToString() == methodName).FirstOrDefault();
```

Comment:
nit: same as above, can simplify to `.FirstOrDefault(condition)`

Reply:
Also implemented as suggested.